core/src/test/java/io/questdb/test/DynamicPropServerConfigurationTest.java

```diff
@@ -295,13 +299,17 @@ public void testPgWireConnectionLimitReload() throws Exception {
                     w.write("pg.net.connection.limit=10\n");
                 }
 
-                TestUtils.assertEventually(() -> Assert.assertEquals(0, metrics.pgWire().connectionCountGauge().getValue()));
+                TestUtils.assertEventually(() -> Assert.assertEquals(0, metrics.pgWireMetrics().connectionCountGauge().getValue()));
+                TestUtils.assertEventually(() -> Assert.assertEquals(2, metrics.pgWireMetrics().getBelowMaxConnectionCountCounter().getValue()));
```

Comment:
nit: should be `getBelowMaxConnectionCountCounter` instead of `belowMaxConnectionCountCounter` for the sake of naming consistency with HTTP metrics:
```suggestion
                TestUtils.assertEventually(() -> Assert.assertEquals(2, metrics.pgWireMetrics().belowMaxConnectionCountCounter().getValue()));
```

Reply:
done in https://github.com/questdb/questdb/pull/5276/commits/4990c10a0889ab93175bb5453854e1e871b440d0